appinventor/components/src/com/google/appinventor/components/annotations/Options.java

```diff
@@ -0,0 +1,26 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2020 MIT, All rights reserved
+// Released under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package com.google.appinventor.components.annotations;
+
+import com.google.appinventor.components.common.OptionList;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation to mark a parameter/return type as accepting an enum value. This should *only* be
+ * used to upgrade old components. New components that want to accept or return an enum should just
+ * use that enum type as the parameter type. E.g:
+ * 
+ * @SimpleProperty
+ * public void FavoriteAnimal (Animal anim) { }
+ */
```

Comment:
For comparison, you should also include an example of *using* the @Option annotation.

Reply:
[Done =)](https://github.com/BeksOmega/appinventor-sources/pull/3/files#diff-f0355e8e44afe002f0977ff65b025ec2R18)

Reply:
Good. I want to resolve this thread, but don't know how to do that ...